### ISSUE TITLE
Inference result and FPS issue

### ISSUE THREAD

**Issue by kungkungpanda** [andy-yun/pytorch-0.4-yolov3#16]
Thanks for your kindness. :-)

I have two questions.

1. 
I have found that there is a difference between the C-based code and the pytorch inference result.
Please see the attached picture.

![result](https://user-images.githubusercontent.com/43112571/45265013-533b2a00-b480-11e8-9b77-2bc63543173a.JPG)

As a result, it is well detected, but especially the box of bike is not the same as the c-based result.
Do you know where this problem occurs in the code?
Solving this difference may help to converge performance to original YOLOv3.
I am trying to resolve this issue.

2.
FPS performance is lower than original one.
The pytorch version seems to be difficult to process in real time.
Is there any way to improve FPS performance?
I think there is not much time difference between pytorch and C in GPU processing...

Thanks for your support.

**Comment by andy-yun** [comment]
1. I am not sure about that. But, in C darknet,  correct_yolo_boxes() is called before return actual detected boxes. I will inspect those part. correct_yolo_boxes() maybe adjust the detected box information.

2. Actually, python version is slower than C version. The python version is used for modeling and conveniency. If you want to fast detection system, you should use C version.

**Comment by andy-yun** [comment]
@kungkungpanda The original C yolo version keeps width/height ration if they use the square sized image. But, in my version and @marvis version, we resized the image to the network size without considering width/height ration. That is the reason that the bounding box is different from each other and our results show the lower performance. I will fix the way to resize the image in the near future. Thanks.

**Comment by andy-yun** [comment]
@kungkungpanda It is a very long time to answer your question. I found some bugs and fixed them. And I modified the source code to keep the original aspect ratio. However, unfortunately, I cannot improve the speed performance. 
Heres the detected image by modified codes.
In my opinion, the result is same to that of original version. 
![predictions](https://user-images.githubusercontent.com/39820731/46957828-59ea3c00-d0d3-11e8-90f0-dc3b40efc62b.jpg)
